### PR TITLE
Slides that build the top half of the final project

### PR DESCRIPTION
**Pull request by SViccari**
Summary:
These changes introduces slides that walk through building the top
section of the final project along with presenting the final HTML and
CSS for students that missed class and may need to copy the code to get
up to speed.

I created an HTML file and CSS file that build the top section of the final project
with the idea that we can point students to GitHub for the code instead of listing
it in the slides. I also added the "final" HTML and CSS in the slides as I haven't
made up my mind yet.

Screen Shot 2019-05-05 at 5 49 15 PM

### CODE REVIEW

File: class4.html [hunk 1]
```diff
@@ -236,7 +239,128 @@ <h2>Set the browser tab Title</h2>
...
+          <h2>Getting Started</h2>
+
+          <ol>
+            <li>Create a generic container using a div.</li>
+            <li>Add a class so we can style it with CSS</li>
+          </ol>
+
+          <pre>
+            <code class="html" data-trim contenteditable>
+              &lt;body&gt;
```
Comment (edwardloveall) [inline class4.html]: Not a big deal but I notice that [later code](https://github.com/SViccari/web-development-intro/pull/13/files#diff-ce81b4db6e311d26883d4c1bf3bf3134R712) don't have to use these HTML entities which is easier to read.

File: class4.html [hunk 2]
```diff
@@ -236,7 +239,128 @@ <h2>Set the browser tab Title</h2>
...
+            <strong>block layout</strong> and <strong>inline layout</strong>.
+            For advanced positioning, we're going to use a third layout option
+            called: <strong>Flexbox</strong>.
+          </p>
+
+          <p>
+            A flexbox layout consists of a <strong>flex container</strong> that
+            holds <strong>flex items</strong>. These <strong>flex items</strong>
+            can "flex" their sizes, growing to fill unused space in the
+            container, or shrinking to avoid overflowing.
```
[the remark this answers is not in the document]
Comment (SViccari) [reply class4.html]: Thanks! I'm pretty sure I borrowed this explanation from a blog post but now I don't recall which one :(

File: class4.html (hunk 1 above)
Comment (SViccari) [reply class4.html]: Unfortunately, if I write `<body>`, Reveal will evaluate the HTML and doesn't show the actual code sample `<body>`. The [docs](https://github.com/hakimel/reveal.js/#code-syntax-highlighting) state "HTML will be escaped by default" so I'd love to be wrong and avoid writing  `&lt;` and `&gt` for `html, head, body` tags but so far, I haven't found another way to write these code samples.

**Comment by SViccari** [comment]
@edwardloveall Thank you for the review 🐳

**Pull request merged**